_sponsors/durthers.md

```diff
@@ -0,0 +1,10 @@
+---
+obj_id: harvey_nash
```

Comment:
This is what links with the `sponsors` field in the event page. So it should be `druthers`